js/todos.js

```diff
@@ -1,37 +1,27 @@
-var App = new (Backbone.View.extend({
+// Initialize global object
+let App = {
     Models: {},
     Views: {},
     Collections: {},
-    template: _.template(
-        '<div class="main-container">' +
-        '<div class="todo-list-container"><h1>ToDo List</h1>' +
-        '<div id="app"></div>' +
-        '</div></div>' 
-    ),
+};
 
-    render: function () {
-        this.$el.html(this.template());
-        return this;
-    },
-    start: function () {
-        this.$el.append(this.render().el);
+$(document).ready(function () {    
 
-        // Initialize some data
-        App.Todos = new App.Collections.Todos();
-        App.Todos.add({ description: 'My First Todo' });
-        App.Todos.add({ description: 'The other Todo' });
-        var initTodosView = new App.Views.TodoList({ collection: App.Todos });
-        initTodosView.render();
-        $('#app').append(initTodosView.el);
+    // Render the main container for the app
+    const appView = new App.Views.Main();
+    $(document.body).append(appView.render().el)
```

Comment:
Generally speaking, in almost all frameworks (Backbone, React, Ember, Vue, etc.) it's best practice to mount into a div that's in the body on boot rather than writing into `document.body`. It's really common that external frameworks and tools inject stuff before the closing body tag and if you're rerendering inside document.body you can kill those scripts/css etc. See here: https://medium.com/@dan_abramov/two-weird-tricks-that-fix-react-7cf9bbdef375